pcsx2/VMManager.cpp

```diff
@@ -3192,14 +3192,29 @@ void VMManager::WarnAboutUnsafeSettings()
 			append(ICON_FA_BUG,
 				TRANSLATE_SV("VMManager", "Debug device is enabled. This will massively reduce performance."));
 		}
+		if (EmuConfig.GS.Dithering == 3)
+		{
+			append(ICON_FA_TV,
+				TRANSLATE_SV("VMManager", "Dithering is set to Force 32 bit. This will break rendering in some games."));
```

Comment:
This isn't just for dithering, basically 16bit draw.

Reply:
Force 32 bit can break rendering that's the point of the warning.